pkg/core/kustomize/kustomize_test.go

```diff
@@ -0,0 +1,156 @@
+package kustomize_test
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/projectriff/riff/pkg/core/kustomize"
+	"io"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+var _ = Describe("Kustomize wrapper", func() {
+
+	var (
+		ownerLabel              *kustomize.Label
+		initialResourceContent  string
+		expectedResourceContent string
+		kustomizer              kustomize.Kustomizer
+		timeout                 time.Duration
+	)
+
+	BeforeEach(func() {
+		ownerLabel = &kustomize.Label{Name: "created-by", Value: "riff"}
+		initialResourceContent = `kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: riff-cnb-cache
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 8Gi`
+		expectedResourceContent = fmt.Sprintf(`apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  labels:
+    %s: %s
+  name: riff-cnb-cache
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 8Gi
+`, ownerLabel.Name, ownerLabel.Value)
+		timeout = 500 * time.Millisecond
+		kustomizer = kustomize.MakeKustomizer(timeout)
+	})
+
+	It("customizes remote resources with provided label", func() {
+		resourceListener, _ := net.Listen("tcp", "127.0.0.1:0")
+		go serve(resourceListener, initialResourceContent)
+		resourceUrl := unsafeParseUrl(fmt.Sprintf("http://%s/%s", resourceListener.Addr().String(), "pvc.yaml"))
+
+		result, err := kustomizer.ApplyLabel(resourceUrl, ownerLabel)
+
+		Expect(err).To(Not(HaveOccurred()))
+		Expect(string(result)).To(Equal(expectedResourceContent))
+	})
+
+	It("customizes local resources with provided label", func() {
+		file, fileUri := localFile("pvc.yaml", initialResourceContent)
+		defer unsafeClose(file)
+
+		resourceUrl := unsafeParseUrl(fileUri)
+
+		result, err := kustomizer.ApplyLabel(resourceUrl, ownerLabel)
+
+		Expect(err).To(Not(HaveOccurred()))
```

Comment:
Isn't `Expect(err).NotTo(HaveOccurred())` is more popular in our other tests? It reads more like English.

Reply:
fixed, thanks for the correction, I never know which one between "to not do" and "not to do" is correct